generate.sh

```diff
@@ -31,9 +31,9 @@ if [ -z "$SOURCE_BUCKET" ]; then
 fi
 
 if [ "$FORCE_GENERATE_ALL" == "true" ]; then
-    python3 docpipeline/__main__.py build-all-docs $SOURCE_BUCKET
+    python3 docpipeline/__main__.py build-all-docs $SOURCE_BUCKET $BUILD_LATEST
```

Comment:
I think I'd prefer to put the logic into the Bash rather than in the Python. What if `BUILD_LATEST` isn't set? What happens with the argument? What if we add an argument after that?

If we go the `LANGUAGE`-as-the-option route from above, perhaps `--language` should be an argument?

Reply:
Will update this logic to be reflected on the bash script rather than on Python as much as possible, depending on which design we move forward with.